pom.xml

```diff
@@ -99,6 +99,7 @@
         <github-api.version>1.111</github-api.version><!-- Used in a Groovy script bellow -->
         <google-auth-library-credentials.version>1.7.0</google-auth-library-credentials.version><!-- TODO: Revert back to using Camel's version when gRPC versions are in sync -->
         <google-oauth-client.version>${google-oauth-client-version}</google-oauth-client.version><!-- TODO: Fix this in Camel https://github.com/apache/camel-quarkus/issues/4139 -->
+        <google-cloud-pubsub.version>1.121.0</google-cloud-pubsub.version><!--Transitively brought by com.google.cloud:libraries-bom -->
```

Comment:
Ideally, we need to find a way to keep this updated. Otherwise we risk drifting from whatever Camel gets from `libraries-bom`.

If that's not possible (or simple), could the exclusions you added be applied instead to the `camel-google-pubsub` dependency?

Reply:
I agree. I tried sync through several boms, but without a success, I'll give it another try. (I might have missed something)

Reply:
I fixed it by syncing through 3 versions (starting with version from camel):

```
 <google-cloud-bom.version>0.182.1</google-cloud-bom.version><!-- @sync com.google.cloud:libraries-bom:${google-cloud-bom-version} dep:com.google.cloud:google-cloud-bom -->
 <google-cloud-pubsub-bom.version>1.121.0</google-cloud-pubsub-bom.version><!-- @sync com.google.cloud:google-cloud-bom:${google-cloud-bom.version} dep:com.google.cloud:google-cloud-pubsub-bom -->
 <google-cloud-pubsub.version>1.121.0</google-cloud-pubsub.version><!-- @sync com.google.cloud:google-cloud-pubsub-bom:${google-cloud-pubsub-bom.version} dep:com.google.cloud:google-cloud-pubsub -->
 ```